src/mocks/data.js

```diff
@@ -149,7 +149,7 @@ export const embeddedDataset = {
       }
     },
   ],
-  files: [
+  study_files: [
     {
       alias: "Test file 1",
       format: "FASTQ",
```

Comment:
The size here in the mock data file is given as a str, but it should be a number.

Important when summing up the sizes.

Reply:
Fixed